sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -346,6 +353,8 @@ def read_all_items(
             feed_options["maxIntegratedCacheStaleness"] = max_integrated_cache_staleness_in_ms
         if hasattr(response_hook, "clear"):
             response_hook.clear()
+        if self.container_link in self.__get_client_container_caches():
+            feed_options["containerRID"] = self.__get_client_container_caches().get(self.container_link).get("_rid")
```

Comment:
how come for the query methods you use `get()` but for the other ones you just access it directly? shouldn't we be consistent?